assets/components/NavBar.js

```diff
@@ -30,50 +44,90 @@ const MenuItems = [
     }
 ]
 
-class NavBar extends Component {
-
-    constructor(){
-        super();
-        this.state = {
-            clicked: false
-        }
-        this.handleClick = this.handleClick.bind(this);
-    }
+function SideDrawer(){
+    const [state, setState] = useState({ right: false });
```

Comment:
Considering using state is similar to global variables, I suggest a more descriptive name than just 'right'

Reply:
Actually, do consider Navbar into a class React component and storing the value in it's state, it might be a better solution. But it's up to you.